packages/usa-site-alert/src/styles/_usa-site-alert.scss

```diff
@@ -6,88 +7,28 @@ $site-alert-icons: (
   emergency: "error",
 );
 
-@mixin site-alert-margins {
-  &:before {
-    left: units($theme-site-margins-mobile-width);
-    @include at-media($theme-site-margins-breakpoint) {
-      left: units($theme-site-margins-width);
-    }
-  }
-}
-
```

Comment:
Removed because it was only referenced once in our system. Let me know if we need to preserve this mixin for project use, etc